### DISCUSSION TITLE
Is this repo going to get its own Wiki?

### DISCUSSION THREAD

**Discussion by kmgallahan** [CommunityToolkit/dotnet#232] [Q&A] [answered]
As per the title, is this repo supposed to get / have its own Wiki?

There are two Wiki references in the main readme, one of which links to the WindowsCommunityToolkit's Wiki, the other just links back to the main readme.

**Comment by michael-hawker** [comment, marked as answer]
@kmgallahan yes, we're actually going to have a consolidated wiki across all repos (with some repo specific bits mixed in). It's on my list of things to do once we get the new Toolkit Labs for Windows setup (so probably ~month-ish after //Build). Basically we'll migrate https://github.com/CommunityToolkit/WindowsCommunityToolkit-wiki to something like https://github.com/CommunityToolkit/wiki and have that drive content towards all the wikis in the individual repos. It'll get generalized for the common bits of our process and access to preview packages (which will be similar) and then we'll probably have individual folders which can provide repo specific pieces like a custom landing page.

FYI @Sergio0694 @brminnick
